comma separated values and documentation

apoorvachikara opened the pull request.

mastermunj commented:
@apoorvachikara Please, do not include package-lock.json file in your commits. That way it won't come in the PR as well.

apoorvachikara commented:
I have removed the package-lock.json. If you check the "file changed" tab - the lock file is removed from there. I removed it long back from the code. Why it is conflicting that the file is not present in the PR?

mastermunj commented:
Okay, so basically you do not need to delete the file. Let the file be there as it is. Make changes in the other files and commit those files in a branch and raise a PR.

apoorvachikara commented:
I made the changes can you check now? Is it fine now?

mastermunj commented:
@apoorvachikara The file is still there. https://github.com/mastermunj/to-words/pull/818/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519

mastermunj commented:
Closing since it's too old!
